lib/matplotlib/tests/test_polar.py

```diff
@@ -481,3 +481,16 @@ def test_polar_neg_theta_lims():
     ax.set_thetalim(-np.pi, np.pi)
     labels = [l.get_text() for l in ax.xaxis.get_ticklabels()]
     assert labels == ['-180°', '-135°', '-90°', '-45°', '0°', '45°', '90°', '135°']
+
+
+@image_comparison(baseline_images=['polar_errorbar'], remove_text=False,
+                  extensions=['png'], style='mpl20')
+def test_polar_errorbar():
+    theta = np.arange(0, 2 * np.pi, np.pi / 8)
+    r = theta / np.pi / 2 + 0.5
+
+    fig = plt.figure(figsize=(10, 10))
```

Comment:
```suggestion
    fig = plt.figure(figsize=(5, 5))
```
Let's try and trim the image size. The image is currently 142kB, which may not seem much, but when we scale this with thousands of tests, we get significant sizes in the repository. We try to keep reference images as small as possible.

Reply:
we could even consider turning the whole axis of `ax.axis("off"). That's sligtly less readable (on doesn't immediately see that 0 is at the top and theta increases clockwise) but eliminates all non-essential parts of the test, making it smaller and less likely to fail due to rendering changes.